### PR TITLE
011 accessibility test

### PR DESCRIPTION
**Pull request by DylanS-03**
added pa11y workflow for accessibility

### CODE REVIEW

File: .github/workflows/localPa11y.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: localPa11yci
```
Comment (JennicaMistry) [inline .github/workflows/localPa11y.yml:1]: Please give this file a proper name.

File: .github/workflows/pa11y.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: pa11yci
```
Comment (JennicaMistry) [inline .github/workflows/pa11y.yml:1]: Please give this a proper name

File: .github/workflows/localPa11y.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - name: Read pa11y_output file.
+      id: pa11y_output
+      uses: juliangruber/read-file-action@v1
+      with:
+        path: ./pa11y_output.txt
+
+    - name: Check for pa11y failures.
+      if: contains(steps.pa11y_output.outputs.content, 'Errors in')
+      run: |
+        echo "::error::The site is failing accessibility tests. Please review the comment in the pull request or the pa11y-ci step in the workflow for details."
```
Comment (JennicaMistry) [inline .github/workflows/localPa11y.yml:42]: Please change the error message as there is no comment left in the PR if the test fails

File: .github/workflows/pa11y.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    - name: Read pa11y_output file.
+      id: pa11y_output
+      uses: juliangruber/read-file-action@v1
+      with:
+        path: ./pa11y_output.txt
+
+    - name: Check for pa11y failures.
+      if: contains(steps.pa11y_output.outputs.content, 'Errors in')
+      run: |
+        echo "::error::The site is failing accessibility tests. Please review the comment in the pull request or the pa11y-ci step in the workflow for details."
```
Comment (JennicaMistry) [inline .github/workflows/pa11y.yml:29]: Please change the error message as there is no comment left in the PR if the test fails

**Review by JennicaMistry — CHANGES_REQUESTED** [review]

**Review by JennicaMistry — APPROVED** [review]

**Review by sunilsehmar78 — APPROVED** [review]
looks fine to me.

**Pull request merged**